gateway-js/src/supergraphManagers/UplinkFetcher/index.ts

```diff
@@ -4,6 +4,7 @@ import resolvable from '@josephg/resolvable';
 import { SupergraphManager, SupergraphSdlHookOptions } from '../../config';
 import { SubgraphHealthCheckFunction, SupergraphSdlUpdateFunction } from '../..';
 import { loadSupergraphSdlFromUplinks } from './loadSupergraphSdlFromStorage';
+export { UplinkFetcherError } from "./loadSupergraphSdlFromStorage"
```

Comment:
It's a little funny to just export the error here for the sake of a simpler re-export from `supergraphManagers/index`.

I think _either_:
* This Error should be defined and exported from here (`UplinkFetcher` owns `UplinkFetcherError`, makes sense to me)
* Or, just skip exporting from here and re-export it from the nested path

One could probably argue that the contents of `loadSupergraphSdlFromStorage.ts` (and transitively `outOfBandReporter.ts`) simply belong in the `UplinkFetcher` class which is probably the root cause of ambiguity for "where should the Error class be defined?", though I'm not asking you to resolve that issue in this PR.

Reply:
Yep. I completely agree. I think it's probably cleaner if I define the Error class in the `index`, and just get it from there in `loadSupergraphSdlFromStorage`.
Will change it. Thanks for the feedback

Reply:
Actually, I was having problems defining `UplinkFetcherError` in the index file, and using it in `loadSupergraphSdlFromStorage`... so I've decided to go with the second option...

However, this makes me wonder, whether the error should belong in a different file (`errors.ts`), and then, export it from there? 

> One could probably argue that the contents of loadSupergraphSdlFromStorage.ts (and transitively outOfBandReporter.ts) simply belong in the UplinkFetcher class which is probably the root cause of ambiguity for "where should the Error class be defined?